build/Build.RetentionPolicy.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Newtonsoft.Json;
+using Nuke.Common;
+using Nuke.Common.Tooling;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+partial class Build : NukeBuild
+{
+    [Parameter("GitHub repository name to look for in the retention policy")]
+    readonly string GitHubRepositoryName = "avivasolutionsnl/sitecore-docker";
```

Comment:
do not add avivasolutions settings to the repo; use a variable in Azure Pipelines